resources/views/moderator/schedule/timeslots-create.blade.php

```diff
@@ -0,0 +1,55 @@
+<x-app-layout>
+    <div>
+        <x-slot name="header">
+            <h2 class="font-semibold text-3xl text-gray-800 dark:text-gray-200 leading-tight">
+                Generate timeslots
+            </h2>
+        </x-slot>
+
+        <div>
+            <form method="POST" action="{{route('moderator.schedule.timeslots.store')}}">
+                @csrf
+                <div class="max-w-7xl mx-auto py-5 mt-12 sm:px-6 lg:px-8 dark:bg-gray-800 bg-gray-200 rounded">
+                    <p class="text-md text-gray-900 dark:text-white">
+                        In order to use the autofill option for the schedule, you need to provide information for the
+                        starting and ending time of the conference.<br>Please note that by starting and ending time of the
+                        conference is meant the starting hour of the first presentation and the ending time of the last
+                        presentation - not including openings/closings.
+                    </p>
+                    <p class="text-md text-gray-900 dark:text-white underline pt-3">Note: There is going to be 10 minutes between the slots so that the
+                                                  participants have time to switch rooms</p>
+                    <div class="col-span-6 sm:col-span-4 pt-5">
+                        <x-label for="starting" value="Starting time"/>
+                        <x-input id="starting" name="starting" type="time" value="{{old('starting')}}"
+                                 class="mt-1 block w-full"/>
+                        <x-input-error for="starting" class="mt-2"/>
+                    </div>
+                    <div class="col-span-6 sm:col-span-4 py-4">
+                        <x-label for="ending" value="Ending time"/>
+                        <x-input id="ending" name="ending" type="time" value="{{old('ending')}}"
+                                 class="mt-1 block w-full"/>
+                        <x-input-error for="ending" class="mt-2"/>
+                    </div>
+                    <hr>
+                    <h3 class="text-lg text-gray-900 dark:text-white pt-5">Lunch break</h3>
+                    <div class="col-span-6 sm:col-span-4 pt-5">
+                        <x-label for="breakStart" value="Starting time of lunch break (default is 12:30)"/>
```

Comment:
When saving but not adding the time for lunchbreak, expected behaviour would be that it saves and then uses said default values. However it errors out with: The break start field must match the format H:i